subt/wifisignal.py

```diff
@@ -3,8 +3,9 @@
 from osgar.node import Node
 
 
-def wifi_scan():
-    with subprocess.Popen('sudo iwlist wlan0 scanning', shell=True,
+def wifi_scan(interface):
+    scanning_cmd = 'sudo iwlist '+interface+' scanning'
```

Comment:
Recently I started to like "formatted strings", i.e. it would look like `scanning_cmd = f'sudo iwlist {interface} scanning'
but this is OK too